identity-server/src/Storage/Models/Client.cs

```diff
@@ -452,7 +451,11 @@ private ICollection<string> Clone()
         private ICollection<string> CloneWith(params string[] values)
         {
             var clone = Clone();
-            foreach (var item in values) clone.Add(item);
+            foreach (var item in values)
+            {
+                clone.Add(item);
+            }
+
```

Comment:
Unrelated, but I'm OK with including it. I wonder why this didn't change in the previous PRs. 🤔

Reply:
I'm running resharper tool to apply changes. It'll occasionally it'll reformat other things in a file so not intentional. I'll revert this to keep the pr clean.